### PR TITLE
AM-2861: Add caseworker-privatelaw-cafcass-cymru to role config

### PR DESCRIPTION
**Pull request by tomxelliott**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/AM-2861

### Change description ###
Add caseworker-privatelaw-cafcass-cymru to role config

> NB: This PR is a dependency of hmcts/am-org-role-mapping-service#1502.

### CODE REVIEW

**Review by prashanthkumarnb — APPROVED** [review]

File: src/main/resources/roleconfig/role_common.json [hunk 1]
```diff
@@ -2001,6 +2001,36 @@
       }
     ]
   },
+  {
+    "name": "caseworker-privatelaw-externaluser-viewonly",
```
Comment (mattnayler) [inline src/main/resources/roleconfig/role_common.json:2005]: Can we add unit tests to `src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/AllServicesOrgRoleTest.java` as we do for other `OTHER_GOV_DEPT` roles such as [dwp, hmrc](https://github.com/hmcts/am-role-assignment-service/blob/50abff261fc17eabba78973dbb3b8daa4dd98320/src/test/java/uk/gov/hmcts/reform/roleassignment/domain/service/drools/AllServicesOrgRoleTest.java#L424-L425).

**Review by mattnayler — CHANGES_REQUESTED** [review]

File: src/main/resources/roleconfig/role_common.json (hunk 1 above)
Comment (mattnayler) [reply src/main/resources/roleconfig/role_common.json:2005]: Also add `listed-hearing-viewer` for PRIVATELAW to unit tests.

Comment (mattnayler) [reply src/main/resources/roleconfig/role_common.json:2005]: Resolved in 37e10b1b96b9927511c31f2457353c826ca1c756

**Review by mattnayler — APPROVED** [review]
LGTM

**Review by prashanthkumarnb — APPROVED** [review]

**Pull request merged**